src/Symfony/Bundle/SecurityBundle/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+5.3.0
+-----
```

Comment:
```suggestion
 
5.3
---
```

Reply:
all lines bellow have 3 digits

Reply:
Fabien documented how to write `CHANGELOG.md` files here, please keep a look and adjust it accordingly.

https://github.com/symfony/symfony-docs/pull/14830

Thanks 🙏

Reply:
I will have a look at it 👍